pkg/sql/schemachanger/scbuild/event_log.go

```diff
@@ -132,6 +132,25 @@ func fullyQualifiedName(b buildCtx, e scpb.Element) string {
 	).FQString()
 }
 
+// ownerName finds the owner of the descriptor that element `e` belongs to.
+// `e` must therefore have a DescID attr.
+func ownerName(b buildCtx, e scpb.Element) string {
```

Comment:
while you're here, could you also add a commit to add eventlog for CREATE FUNCTION? thanks!